bin/run-versioned-tests.sh

```diff
@@ -21,5 +21,14 @@ fi
 
 export AGENT_PATH=`pwd`
 
+# I think we're skipping koa because a new version was breaking tests
```

Comment:
@koa/router 8.03+ breaks segment naming for nested routes.

We need to prob adjust the versions in koa module and publish to limit until fixed (which unfortunately is not straightforward). We might update the comment to that to help future us. Can be a follow-up.